### PR TITLE
VBM-51: welcome screen layout

### PR DESCRIPTION
**Pull request by mazurGit**

### CODE REVIEW

File: packages/mobile/src/components/components.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-export { Text, View } from 'react-native';
+export { Text, View, Image } from 'react-native';
 export { SafeAreaView } from 'react-native-safe-area-context';
 export { ScreenWrapper } from './screen-wrapper/screen-wrapper';
 export { Button } from './button/button';
 export { Input } from './input/input';
 export * from './icons/icons';
 export { Spinner } from './spinner/spinner';
+export { LinearGradient } from './linear-gradient/linear-gradient';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/components/components.ts:8]:
```suggestion
export { default as LinearGradient } from 'react-native-linear-gradient';
```

File: packages/mobile/src/screens/home/components/product.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { styles } from './style';
+
+type Props = {
+  title: string;
+  description: string;
+  price: number;
+  imageUrl?: string;
+};
+
+export const Product: FC<Props> = ({ title, description, price, imageUrl }) => {
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/home/components/product.tsx:14]: I'd suggest extracting image of stamp and dedicating this component for one specific purpose -- displaying the lot card.
Also if you making just mocked card there is no sense in passing props, you can place text directly here.

File: packages/mobile/src/screens/home/home.tsx [hunk 3]
```diff
@@ -1,11 +1,55 @@
...
+      start={{ x: 0, y: 0.35 }}
+      end={{ x: 0, y: 0.65 }}
+      colors={[ColorPalette.BLUE_100, ColorPalette.YELLOW_200]}
+      style={styles.wrapper}
+    >
+      <Image source={images.logo_small} style={styles.logo} />
+      <View style={styles.stamp_wrapper}>
+        <Product
+          title='Mark "Russian Warship"'
+          description="The stamp was made by Ukrposhta in 2022. The expression became famous..."
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/home/home.tsx:31]: These should also be translated

File: packages/mobile/src/assets/pictures/images/images.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+const images = {
+  logo_small: require('~/assets/pictures/images/logo_small.png'),
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/assets/pictures/images/images.ts:2]: Why complicating folder structure? We can keep all images within assets/images

File: packages/mobile/src/localization/languages/ua.ts [hunk 5]
```diff
@@ -1,4 +1,10 @@
 export const ua = {
+  welcome: {
+    TITLE: 'Марка "Русский военный корабль"',
+    DESCRIPTION: 'Марка була випущена Укрпоштою у 2022. Вислів став відомим...',
+    BET: 'Ставка',
+    CURRENCY_UA: 'ГРН',
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/localization/languages/ua.ts:6]:
```suggestion
    UAH: 'ГРН',
```

**Review by AleksandrTermenzhy — APPROVED** [review]

File: packages/mobile/src/components/components.ts [hunk 6]
```diff
@@ -1,8 +1,9 @@
-export { View } from 'react-native';
+export { Image, View } from 'react-native';
 export { SafeAreaView } from 'react-native-safe-area-context';
 export { ScreenWrapper } from './screen-wrapper/screen-wrapper';
 export { Button } from './button/button';
 export { Input } from './input/input';
 export { Text } from './text/text';
 export * from './icons/icons';
 export { Spinner } from './spinner/spinner';
+export { default as LinearGradient } from 'react-native-linear-gradient';
```
Comment (seal323) [inline packages/mobile/src/components/components.ts:9]: move import from library to the top for keeping the structure

File: packages/mobile/src/screens/welcome/components/product/style.ts [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  wrapper: {
+    position: 'absolute',
+    borderRadius: 4,
+    right: -15,
+    bottom: -50,
+    width: '55%',
+    backgroundColor: 'rgba(255, 255, 255, 0.9)',
```
Comment (seal323) [inline packages/mobile/src/screens/welcome/components/product/style.ts:11]: All colors should be used from ```ColorPalette```

File: packages/mobile/src/screens/welcome/components/product/product.tsx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+          >
+            {`360 ${t('welcome.CURRENCY_UA')}`}
+          </Text>
+          <Button
+            label={t('welcome.BET')}
+            type={ButtonType.SECONDARY}
+            view={ButtonAppearance.FILLED}
+            fontSize={11}
+            onPress={() => {
+              // to do
```
Comment (seal323) [inline packages/mobile/src/screens/welcome/components/product/product.tsx:39]: Comments should be standardized //TODO:

**Review by seal323 — APPROVED** [review]

**Pull request merged**